datalad/interface/common_cfg.py

```diff
@@ -21,4 +25,52 @@
     #            'text': 'Eh?'}),
     #    'destination': 'dataset',  # where it should be stored by default
     #}
+    'datalad.locations.cache': {
+        'ui': ('question',
+               {'title': 'Cache directory',
+                'text': 'Where should datalad cache files?'}),
+        'destination': 'global',
+        'default': dirs.user_cache_dir,
+    },
+    # this is actually used in downloaders, but kept cfg name original
+    'datalad.crawl.cache': {
+        'ui': ('yesno', {
+               'title': 'Crawler download caching',
+               'text': 'Should the crawler cache downloaded files?'}),
+        'destination': 'local',
+        'type': bool,
+    },
+    'datalad.crawl.dryrun': {
+        'ui': ('yesno', {
+               'title': 'Crawler dry-run',
+               'text': 'Should the crawler ... I AM NOT QUITE SURE WHAT?'}),
+        'destination': 'local',
+        'type': EnsureBool(),
+    },
+    'datalad.externals.nda.dbserver': {
+        'ui': ('question', {
+               'title': 'NDA database server',
+               'text': 'Hostname of the database server'}),
+        'destination': 'global',
```

Comment:
You can. Just add a 'default' field. But in this case the default comes from the nda code call, because that is where it used to be. Both is possible.